.travis.yml

```diff
@@ -37,11 +38,13 @@ jobs:
         - chmod +x operator-sdk
         - mv operator-sdk $GOPATH/bin/
 
+        # Download OPM tool
```

Comment:
what is OPM tool? any references?

Reply:
Taken from https://github.com/operator-framework/operator-registry: `opm generates and updates registry databases as well as the index images that encapsulate them.`

It's THE tool used to build the index image in `build-bundle-index-image` target